modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -261,11 +276,13 @@ public function render() {
    * @param string $iiif_base_id
    *   The URL for the request, minus the last part of the URL,
    *   which is likely "manifest".
+   * @param \Drupal\taxonomy\TermInterface|null $structured_text_term
+   *   The term that structured text media references, if any.
    *
    * @return array
    *   List of IIIF URLs to display in the Openseadragon viewer.
    */
-  protected function getTileSourceFromRow(ResultRow $row, $iiif_address, $iiif_base_id) {
+  protected function getTileSourceFromRow(ResultRow $row, $iiif_address, $iiif_base_id, $structured_text_term) {
```

Comment:
`$structured_text_term` isn't used here? Bad merge conflict resolution, since the move to/introduction of `::getStructuredTextTerm()`?

Reply:
Done, as always thanks for being so thorough.